完善模型资源相关 (#1742)
cocos-creator/creator-docs: en/editor/publish/build-options.md

If this option is not checked, release mode will be built and the editor will compress and obfuscate the `uuid` of the asset, the engine script and project script generated by the build, and subpackage the `json` of the same type asset, reducing the times of asset loading.

> If this option is checked, debug mode will be built, allowing you to debug the project and easily locate problems.

Reviewer: ```suggestion
If this option is checked, debug mode will be built, allowing the project to be debugged which easily allows locating of problems.
```
